src/plugins/profiling/scripts/import_from_testing.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Exit on use of undefined variables and on command failures.
+set -eu
+
+die() {
+    if [[ "$@" ]]; then
+        echo -e "$@\n" >&2
+    fi
+    exit 1
+}
+
+usage() {
+  die "Usage: $0 <files>\n" \
+  "\n" \
+  "The data files are imported into the local ES instance (localhost:9200).\n" \
+  "The naming convention is that the first part (before -data...) is the index name.\n"
+  "\n" \
+  "You need to put the credentials into the ELASTIC_LOCAL_CREDENTIALS env var (...=\"user:pw\")"
+}
+
+import() {
+  local index=$1
+  local path=$2
+  local dir=${path%$file}
+
+  if [[ "$dir" == "" ]]; then
+    dir="."
+  fi
+
+  pushd "$dir"
+  docker run --rm -ti --net=host -v "$PWD:/data" elasticdump/elasticsearch-dump:latest \
+    --input="/data/$file" \
+    --output="http://admin:changeme@localhost:9200/$index" \
+    --type=data --fsCompress --noRefresh --support-big-int --limit=10000
+  popd
+}
```

Comment:
Did we agree on adding the mappings too when exporting and re-importing?
Will you be adding that in this PR?

Reply:
I'd say we should create another PR. Adding mappings has some edges that needs to be thought of. And except for my stupid mistake today, we possibly don't need them right now.